docs/sources/srv-record.md

```diff
@@ -0,0 +1,29 @@
+# SRV record with CRD source
+
+You can create and manage SRV records with the help of [CRD source](../contributing/crd-source.md)
+and `DNSEndpoint` CRD. The implementation of this feature depends on provider API support, this feature is currently known to be supported supported by `akamai`, `civo`, `cloudflare`, `ibmcloud`, `linode`, `rfc2136` and `pdns` providers.
```

Comment:
This might not be completely accurate but I can't test every provider, I just built this list by grepping for mentions of `SRV` in the providers folder and searching the issue tracker.